src/cdk-experimental/menu/menu-bar.ts

```diff
@@ -25,6 +25,21 @@ import {CDK_MENU, Menu} from './menu-interface';
 import {CdkMenuItem} from './menu-item';
 import {MenuStack, MenuStackItem, FocusNext} from './menu-stack';
 
+/**
+ * Check if the given element is part of the cdk menu module.
+ * @param target the element to check.
+ * @return true if the given element is part of the menu module.
+ */
+function isCdkMenuComponent(target: Element) {
```

Comment:
How about `isElementWithinMenu`?

Reply:
It's not really checking if the element is **within** a menu, but rather if the element is a menu its self. If we go with the below suggestions it could be `isMenu`. Thoughts?

Reply:
Yeah, good point, maybe just `isMenuElement`